### PR TITLE
Changelog for ACS Calling beta.12 release

### PR DESCRIPTION
**Pull request by raosanat**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,43 @@
 ### Key Bug Fixes
 **SERVICE**
 
+## 1.0.0-beta.12 (2021-04-13)
```
Comment (JoshuaLai) [inline CHANGELOG.md:14]: ## 1.0.0-beta.12 (UNRELEASED) 
Is still set up at line 3. 
I think you should bump it up to .13

**Review by JoshuaLai — CHANGES_REQUESTED** [review]
One nit pick commnet

File: CHANGELOG.md (hunk 1 above)
Comment (raosanat) [reply CHANGELOG.md:14]: you mean the `1.0.0-beta.12` to `1.0.0-beta.13` ?

Comment (JoshuaLai) [reply CHANGELOG.md:14]: No not here. In line 3 of this file it would need to be updated 
`## 1.0.0-beta.13 (UNRELEASED)`

Comment (raosanat) [reply CHANGELOG.md:14]: ahh i see what you mean now :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,43 @@
 ### Key Bug Fixes
 **SERVICE**
```
Comment (tjprescott) [inline CHANGELOG.md:12]: Remove lines 3-12

**Review by JoshuaLai — APPROVED** [review]

**Review by tjprescott — APPROVED** [review]

**Pull request merged**